File: testdata/bench/go/BenchmarkTest00061.go
package bench

import (
	"io"
	"net/http"
)

// SSRF via http.Get
func Handler00061(w http.ResponseWriter, r *http.Request) {
	target := r.URL.Query().Get("url")
	resp, _ := http.Get(target)
	defer resp.Body.Close()
	io.Copy(w, resp.Body)
}

File: testdata/bench/rust/BenchmarkTest00061.rs
use actix_web::{web, HttpResponse};
use std::collections::HashMap;

// Vulnerable: SSRF via direct reqwest::get with user URL
async fn handler(query: web::Query<HashMap<String, String>>) -> HttpResponse {
    let param = query.get("url").unwrap_or(&String::new()).clone();
    let resp = reqwest::get(&param).await.unwrap();
    HttpResponse::Ok().body(resp.text().await.unwrap_or_default())
}
